packages/sysutils/busybox/system.d/storage-log.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Create Persistent Log Directory on /storage
+DefaultDependencies=no
+RequiresMountsFor=/storage
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/mkdir -p /storage/.log/journal
```

Comment:
Would it be "cleaner" to locate this directory within `/storage/.cache`? The `.cache` and `.config` directories are under our control, whereas I tend to feel that `/storage` is not (ie. increased risk of clashing with an existing directory).

Reply:
yes, makes sense - I'll change it to `/storage/.cache/log`